assets/src/components/amp-support/error-sources.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * To render error source information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for error source data.
+ */
+export function ErrorSources( { data: errorSources } ) {
+	if ( 'object' !== typeof errorSources ) {
+		return null;
+	}
+
+	return (
+		<details open={ false }>
+			<summary>
+				{ __( 'Error Sources', 'amp' ) }
+				{ ` (${ errorSources.length || 0 })` }
```

Comment:
An empty array's length equals 0 so there's no need for the OR operator. Also, the entire title should be localizable, i.e.:

```suggestion
				{ sprintf(
					/* translators: Placeholder is the number of error sources */
					__( 'Error Sources (%d)', 'amp' ),
					data.errorSources.length,
				) }
```